docs/client/java/partials/java_metrics.md

```diff
@@ -0,0 +1,65 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info
+This feature is available in OpenLineage 1.11 and above
+:::
+
+OpenLineage integration for Spark increases observability of Spark jobs.
```

Comment:
Can we remove `spark` from this sentences? This makes sense in general too and this section should not refer to specific integration. Integration specific details can be put into tab items.

Reply:
Fixed.